src/store/modules/facility/actions.ts

```diff
@@ -160,6 +162,35 @@ const actions: ActionTree<FacilityState, RootState> = {
       facilityTypeId: ''
     })
     commit(types.FACILITY_LIST_UPDATED , { facilities: [], total: 0 });
+  },
+
+  async getFacilityParties({ commit }, payload) {
+    let parties = []
+    const params = {
+      inputFields: {
+        facilityId: payload.facilityId
+      },
+      entityName: "FacilityAndParty",
+      filterByDate: 'Y',
+      orderBy: 'partyId DESC',
+      fieldList: ['facilityId', 'firstName', 'fromDate', 'lastName', 'groupName', 'partyId', 'roleTypeId'],
+      viewSize: 100
+    }
+
+    try {
+      const resp = await FacilityService.getFacilityParties(params)
+
+      if(!hasError(resp) && resp.data.count) {
+        parties = resp.data.docs
```

Comment:
Create name for the party here itself, so we don't have to define the same logic of checking groupName.